README.md

```diff
@@ -58,10 +58,10 @@ file in the root of the source tree to see what version you're on. If you're on
 the master development branch it will just say 'devel'.
 
 #### Install the Zephyr SDK
-Download the [latest Zephyr SDK] (https://www.zephyrproject.org/downloads/tools), then:
+Download the [latest Zephyr SDK] (https://www.zephyrproject.org/downloads), then:
 ```bash
-$ chmod +x /path/to/zephyr-sdk-<VERSION>-i686-setup.run
-$ sudo /path/to/zephyr-sdk-<VERSION>-i686-setup.run
+$ chmod +x /path/to/zephyr-sdk-<VERSION>-setup.run
+$ sudo ./path/to/zephyr-sdk-<VERSION>-setup.run
```

Comment:
Remove the '.' here. The idea of /path/to is it means whatever path the user saved to, not necessarily beneath the current directory. Otherwise looks good, thanks.